src/main/java/tech/pegasys/pantheon/plugin/services/storage/KeyValueStorageTransaction.java

```diff
@@ -23,7 +23,7 @@
    * <p>If a previously value had been store against the given key, the old value is replaced by the
    * given value.
    *
-   * @param key the given value is to be associated with.
+   * @param key the given value is to be associated with, never <code>null</code>.
```

Comment:
and then I don't think we need the never null here.  I think we settled on not adding them all over the place?

Reply:
🤷‍♂ I'll removing it then